WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SiteCreationService.java

```diff
@@ -128,6 +131,7 @@ static Notification progress(Context context, int progress, @StringRes int title
 
         static Notification success(Context context) {
             return AutoForegroundNotification.success(context,
+                                                      NotificationsUtils.GENERAL_NORMAL_CHANNEL_ID,
```

Comment:
I'm wondering whether we should use `GENERAL_IMPORTANT_CHANNEL_ID` for both site creation success and failure?

Reply:
We could! Just a general comment: this PR does not change the logic we already had in place (i.e. the same importance is kept normal for notifications that were already using the normal priority, and only high for notifications that were already using the high priority). 
That said, I don't mind changing it to `GENERAL_IMPORTANT_CHANNEL_ID` if you feel strongly about it 👍

Reply:
Nope, no strong feelings about that. I'm ready to merge but looks like there are some checkstyle issues causing Travis to fail.

Reply:
oops - just fixed that in b55658d and restarted Travis 👍

Reply:
Travis is still failing, but I'm not sure why.